### ISSUE TITLE
Can linear_water, area_water, and other features accept list of counties?

### ISSUE THREAD

**Issue by binkleym** [walkerke/tigris#97]
Can the linear_water, area_water, and other features be modified to pull geometry for lists of counties instead of individual counties?   For instance, I would like to be able to do something like:

### Tennessee
my_states <- c("47")

### Nashville Metropolitan Statistical Area
my_counties <- c("015", "021", "037", "043", "111", "119", "147", "149", "159", "165", "169", "187", "189")

### What I'd like to do...
rivers <- linear_water(my_states, my_counties, year = 2018, class = "sf") 

### What I get when I try:
Error in unzip(file_loc, exdir = cache_dir, overwrite = TRUE) : 
  invalid zip name argument
In addition: Warning messages:
1: In if (grepl("^[[:digit:]]+$", county)) { :
  the condition has length > 1 and only the first element will be used
2: In if (county %in% county_table$county_code) { :
  the condition has length > 1 and only the first element will be used
3: In if (refresh | !file.exists(file_loc)) { :
  the condition has length > 1 and only the first element will be used
4: In if (refresh | !file.exists(file.path(cache_dir, sprintf("%s.shp",  :
  the condition has length > 1 and only the first element will be used

**Comment by walkerke** [comment]
Yes I can look into this.  Until it is implemented you can do: 

```r
library(tigris)
library(purrr)
options(tigris_use_cache = TRUE)

my_counties <- c("015", "021", "037", "043", "111", "119", "147", "149", "159", "165", "169", "187", "189")

tn_rivers <- map(my_counties, ~{
  linear_water("TN", .x, class = "sf")
}) %>%
  rbind_tigris()

plot(tn_rivers$geometry)
```
![image](https://user-images.githubusercontent.com/4872833/86912637-f42db700-c0e2-11ea-86d6-cb98fa79e5a8.png)

**Comment by walkerke** [comment]
This is now implemented and will be included in tigris 1.0, coming next week.

**Comment by binkleym** [comment]
Thank you so much!
